app/serializers/basic_user_serializer.rb

```diff
@@ -1,5 +1,13 @@
 class BasicUserSerializer < ApplicationSerializer
-  attributes :id, :username, :avatar_template
+  attributes :id, :username, :name, :avatar_template
+
+  def name
+    if Hash === object
```

Comment:
Good idea! I made the change but instead of `user&.name` I did `user.try(:name)` because there is a case where user is not nil but the object is of `PostAction` which does not contain `name`.

Reply:
> I did user.try(:name) because there is a case where user is not nil but the object is of PostAction which does not contain name.

Hmm it sounds like a bug that the `object` is a `PostAction` since the `UserSerializer` is meant to only serialize `User` objects.

Reply:
Looks like it.. here is the failing spec that made me realize this:

```
PostActionUsersController with render always allows you to see your own actions
 Failure/Error: expect(response.status).to eq(200)

   expected: 200
        got: 500

   (compared using ==)
 # ./spec/requests/post_action_users_controller_spec.rb:14:in `block (3 levels) in <top (required)>'
 # ------------------
 # --- Caused by: ---
 # NoMethodError:
 #   undefined method `name' for #<PostAction:0x007fd3c553b8a8>
 #   ./app/serializers/basic_user_serializer.rb:8:in `name'
```

Will investigate more about this case.